roles/pulp_devel/templates/alias.bashrc.j2

```diff
@@ -1,11 +1,23 @@
 # If adding new functions to this file, note that you can add help text to the function
 # by defining a variable with name _<function>_help containing the help text
 
-SERVICES=("pulpcore-content {% for num in range(pulp_workers) %}pulpcore-worker@{{ num + 1 }} {% endfor %}pulpcore-resource-manager pulpcore-api")
+SERVICES=$(systemctl --type=service | grep pulpcore | cut -d' ' -f1)
+ALL_SERVICES=$(systemctl --type=service --all | grep pulpcore | cut -d' ' -f1)
+
+_pulpcore_version() {
+    workon pulp
+    pip show pulpcore | grep Version | cut -d' ' -f2
+}
 
 _paction() {
-    echo systemctl $@ ${SERVICES}
-    sudo systemctl $@ ${SERVICES}
+    Y_VERSION=$(_pulpcore_version | cut -d'.' -f2)
+    if [ "$Y_VERSION" -lt 16 ]; then
+        echo systemctl $@ ${ALL_SERVICES}
+        sudo systemctl $@ ${ALL_SERVICES}
+    else
+        echo systemctl $@ ${SERVICES}
+        sudo systemctl $@ ${SERVICES}
+    fi
```

Comment:
the idea is starting 'pulpcore-resource-manager' easily in dev environment when you need to use other pulpcore versions

Reply:
Maybe it's time to say: "Use and old branch of the installer if you know how to use an old branch of pulpcore."